dashboard/slurm_scripts/compute_stats_on_pseudo_crawl.slurm

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+#SBATCH --job-name=pseudo_crawl_compute_stats_v5
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=1          # crucial - only 1 task per dist per node!
+#SBATCH --cpus-per-task=4         # number of cores per tasks
+#SBATCH --hint=nomultithread         # we get physical cores not logical
+#SBATCH --partition=cpu_p1
+#SBATCH --time 10:00:00              # maximum execution time (HH:MM:SS)
+#SBATCH --output=/gpfswork/rech/six/uty16tp/code/big_science/logs/compute_stats_v5/%x-%j.out           # output file name
+#SBATCH --array=1-604
```

Comment:
You need to add the new seeds. And I need to update the csv with the new seeds ....